tests/tools/onert_train/src/args.h

```diff
@@ -59,6 +59,7 @@ class Args
   const int getLossType(void) const { return _loss_type; }
   const int getLossReductionType(void) const { return _loss_reduction_type; }
   const int getOptimizerType(void) const { return _optimizer_type; }
+  const int getMetricType(void) const { return _metric_type; }
```

Comment:
(Optional) Returning `const int` value is meaningless.
```suggestion
  int getMetricType(void) const { return _metric_type; }
```

Reply:
Let's change this after merging this PR.